domain/schema/controller.go

```diff
@@ -347,3 +349,79 @@ BEGIN
     VALUES (4, 2, OLD.controller_id, DATETIME('now'));
 END;`[1:]
 }
+
+func modelMigrationSchema() string {
+	return `
+CREATE TABLE model_migration (
+	uuid					TEXT PRIMARY KEY,
+	attempt					INT,
+	target_controller_uuid	TEXT NOT NULL,
+	target_cacert			TEXT,
+	target_entity			TEXT,
+	target_password			TEXT,
+	target_macaroons		TEXT,
+	active					BOOLEAN,
+	start_time				TIMESTAMP,
+	success_time			TIMESTAMP,
+	end_time				TIMESTAMP,
+	phase					TEXT,
+	phase_changed_time		TIMESTAMP,
+	status_message			TEXT,
+	-- target_controller_alias
+	-- target_addresses
+	CONSTRAINT				fk_model_migration_target_controller
```

Comment:
Given that you have this relational constraint, this table no longer needs:
- `target_cacert`
- `target_controller_alias`
- `target_addresses`